src/main/java/com/cgessinger/creaturesandbeasts/entities/CindershellEntity.java

```diff
@@ -1,10 +1,7 @@
 package com.cgessinger.creaturesandbeasts.entities;
 
 import com.cgessinger.creaturesandbeasts.containers.CinderFurnaceContainer;
-import com.cgessinger.creaturesandbeasts.init.CNBBlocks;
-import com.cgessinger.creaturesandbeasts.init.CNBEntityTypes;
-import com.cgessinger.creaturesandbeasts.init.CNBItems;
-import com.cgessinger.creaturesandbeasts.init.CNBSoundEvents;
+import com.cgessinger.creaturesandbeasts.init.*;
```

Comment:
Please don't convert my imports to a wildcard. I consciously avoid wildcards.

Reply:
Oh apologies, I think my IDE may have done it automatically